delocate/cmd/delocate_fuse.py

```diff
@@ -1,42 +1,25 @@
 #!/usr/bin/env python3
 """Fuse two (probably delocated) wheels.
 
-Overwrites the first wheel in-place by default.
+Command is no longer available. To fuse two wheels together use
+'delocate-merge'. NOTE: 'delocate-merge' does not overwrite the first wheel. It
+creates a new wheel with an automatically determined name. If the old behavior
+is needed (not recommended), pin the version to 'delocate==0.11.0'.
 """
 
 # vim: ft=python
-from __future__ import absolute_import, division, print_function
-
-from argparse import ArgumentParser
-from os.path import abspath, basename, expanduser
-from os.path import join as pjoin
-
-from delocate.cmd.common import common_parser, verbosity_config
-from delocate.fuse import fuse_wheels
-
-parser = ArgumentParser(description=__doc__, parents=[common_parser])
-parser.add_argument(
-    "wheels", nargs=2, metavar="WHEEL", type=str, help="Wheels to fuse"
-)
-parser.add_argument(
-    "-w",
-    "--wheel-dir",
-    action="store",
-    type=str,
-    help="Directory to store delocated wheels"
-    " (default is to overwrite 1st WHEEL input with 2nd)",
-)
+from __future__ import annotations
 
 
 def main() -> None:  # noqa: D103
-    args = parser.parse_args()
-    verbosity_config(args)
-    wheel1, wheel2 = [abspath(expanduser(wheel)) for wheel in args.wheels]
-    if args.wheel_dir is None:
-        out_wheel = wheel1
-    else:
-        out_wheel = pjoin(abspath(expanduser(args.wheel_dir)), basename(wheel1))
-    fuse_wheels(wheel1, wheel2, out_wheel)
+    print(
+        "'delocate-fuse' is no longer available. To fuse two wheels together"
+        " use 'delocate-merge'. NOTE: 'delocate-merge' does not overwrite the"
+        " first wheel. It creates a new wheel with an automatically determined"
+        " name. If the old behavior is needed (not recommended), pin the"
+        " version to 'delocate==0.11.0'."
+    )
+    return 1
```

Comment:
The return value from `main` isn't used by the caller. Simplest solution is to raise `SystemExit`.
```suggestion
    raise SystemExit(1)
```

Reply:
But this is what the entry point script looks like:
```python
if __name__ == '__main__':
    sys.argv[0] = re.sub(r'(-script\.pyw|\.exe)?$', '', sys.argv[0])
    sys.exit(main())
```
The return value from main is used as the exit code. `sys.exit` raises `SystemExit`.

Reply:
You're right. I was looking at the scripts own call of `main`, not the entry point code.

The command scripts themselves are not set up correctly to return `int` unless they're modified. Raising `SystemExit` does the same without any other changes.